src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.querySelector('tr');
+
+rows.forEach((row) => {
+  const cells = row.querySelector('td, th');
```

Comment:
The `querySelector` method will only select the first 'td' or 'th' element in each row. If you intend to select all cells, you should use `querySelectorAll` instead.